Simperium/src/main/java/com/simperium/client/ChannelProvider.java

```diff
@@ -20,4 +20,9 @@
     public int getLogLevel();
 
 
+    public interface HeartbeatListener {
+        public void onBeat();
+    }
+
+    public void addHeartbeatListener(HeartbeatListener listener);
```

Comment:
Since `addHeartbeatListener(HeartbeatListener listener)` was added to `ChannelProvider`, we'll need to add it to `MockChannelProvider` too.  `MockChannelProvider` is used indirectly for testing.  Testing isn't utilized very much, but we'll at least need to add it to pass the continuous integration environment.